src/tasks/tempMute.js

```diff
@@ -0,0 +1,31 @@
+const { Task } = require("klasa");
+const ModLog = require("../../lib/structures/ModLog");
+
+module.exports = class extends Task {
+
+    async run({ guildID, userID }) {
+        const guild = this.client.guilds.get(guildID);
+        const member = await guild.members.fetch(userID).catch(() => null);
+        if (!guild || !member) return;
+
+        const roleID = guild.settings.permissions.mutedRole;
+        const role = await guild.roles.get(roleID);
+
+        if (!role) return;
+        const myRole = guild.me.roles.find(r => r.managed);
```

Comment:
What is this for

Reply:
To check if PenguBot's role is in a lower hierarchy than the role it's trying to assign.

Reply:
thats not how.

Reply:
Then you go ahead and tell me how?

Reply:
Go lookup what Role#managed is.